packages/design-system/src/components/KBar/KBar.styles.ts

```diff
@@ -0,0 +1,47 @@
+import { css, darkTheme } from '@jeromefitz/design-system/src/stitches.config'
```

Comment:
Please reference these files locally.

🤔  though maybe that is the way to do circular path mapping 😆